spec/ask_export/csv_splitter_spec.rb

```diff
@@ -0,0 +1,83 @@
+require "active_support/testing/time_helpers"
+
+RSpec.describe AskExport::CsvSplitter do
+  include ActiveSupport::Testing::TimeHelpers
+
+  describe ".call" do
+    around do |example|
+      expect { example.run }.to output.to_stdout
+    end
+
+    around do |example|
+      travel_to(Time.parse("2020-05-01 11:00 UTC")) { example.run }
+    end
+
+    before do
+      allow(File).to receive(:write)
+    end
+
+    let(:live_export_path) do
+      File.join(__dir__, "../support/files/live_export.csv")
```

Comment:
I found it confusing to see the paths to real files, and then to stub the output of the thing that's processing them. Seeing this here, I was looking for where the loaded data pops out in the tests. Could we just use a `/a-path.csv`?

Reply:
Ahh, now I see: it's converting the loaded data for the builder. Could we separate those concerns? I realise we're now testing two things here: that we're presenting the correct data to the builder; and that we're writing files for the end result.

Reply:
Yup I've broken down these tests a bit.